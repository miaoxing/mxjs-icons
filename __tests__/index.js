import React from 'react';
import Icon from '..';
import {render} from '@testing-library/react';

describe('Icon', () => {
  test('basic', () => {
    const result = render(<Icon type="text"/>);
    expect(result.container).toMatchSnapshot();
  });
});
