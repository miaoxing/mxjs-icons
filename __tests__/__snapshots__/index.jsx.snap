// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon basic 1`] = `
<div>
  <span
    class="anticon"
    role="img"
  >
    <svg
      aria-hidden="true"
      class=""
      fill="currentColor"
      focusable="false"
      height="1em"
      width="1em"
    >
      <use
        xlink:href="#text"
      />
    </svg>
  </span>
</div>
`;
